ui/hooks/useNftCollectionsMetadata.ts

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useMemo, useState } from 'react';
+import { NameType } from '@metamask/name-controller';
+import { Collection } from '@metamask/assets-controllers';
+import { useSelector } from 'react-redux';
+import { getCurrentChainId } from '../selectors';
+import { fetchNftCollectionsMetadata } from '../store/actions';
+
+export type UseNftCollectionsMetadataRequest = {
+  value: string;
+  type: NameType;
```

Comment:
Do we use the `type` anywhere?

Reply:
Done https://github.com/MetaMask/metamask-extension/pull/25692/commits/a239cf5b0a5ca0a7a518658cc3706d60100f8626